### ISSUE TITLE
Won't install in Safari 13

### ISSUE THREAD

**Issue by ghost** [deathau/markdownload#76]
Hi,
Safari extension won't install. 
It gets as far as 
![Screen Shot 2021-04-19 at 10 46 25 pm](https://user-images.githubusercontent.com/40692411/115238486-1586fd80-a161-11eb-914f-1d4848caba89.png)

and clicking that big blue button does nothing

**Comment by kTag** [comment]
Same issue. Seems nobody is following these issues reported... Should I ask for refund?

**Comment by deathau** [comment]
I have done some digging into this, and see https://github.com/deathau/markdownload/issues/7#issuecomment-865489058 for the full response, but the tl;dr is it seems like a Big Sur issue and I can't upgrade on my equipment to debug further.

@kTag you're well within your rights to request a refund (you bought a thing and it doesn't work!), though I don't actually know how / if that's possible through the Apple App store, and I'll have to look into that also.

**Comment by kTag** [comment]
Thanks for the quick reply. 

So you mean it’s really a Safari issue right?

Let me try to see if it works for other browsers. I don’t see any issue trying to use a dedicated browser for that extension.

**Comment by deathau** [comment]
So I now have a new mac mini with Big Sur installed 🎉 
I can replicate the issue with the big blue button doing nothing, but the extension still installed correctly. Opening Safari prompted me to enable it, it asked for permissions, and everything seemed to work fine.
I then _uninstalled_ it and re-installed, and it didn't prompt me for anything in safari, but it was all enabled from the previous install. I just had to add it manually from the extension preferences.
I took a screen recording of that second time to demonstrate, but I used QuickTime, and the file is > 10MB so I can't upload it here...
